packages/synapse-react-client/src/lib/containers/discussion_forum/DiscussionReply.tsx

```diff
@@ -52,75 +51,69 @@ export const DiscussionReply: React.FC<DiscussionReplyProps> = ({
       ) : (
         <>
           {message && (
-            <div>
-              <UserCard
-                withAvatar={true}
-                avatarSize="MEDIUM"
-                showCardOnHover={true}
-                size={SMALL_USER_CARD}
-                ownerId={reply.createdBy}
-              />
-              <div className="message-body">
-                <MarkdownSynapse
-                  markdown={message}
-                  objectType={ObjectType.REPLY}
+            <>
```

Comment:
I added the outer `<>` here so that I could move the ForumEditorThread and WarningModal within this section, so they are only rendered after the thread data is loaded, so the thread title and thread body are pre-filled appropriately. Most of the changes in the diff are indentation differences